USAGE.md

```diff
@@ -110,6 +110,19 @@ When choosing which source to use for a citation, we recommend the following ord
 9. For references that do not have any of the above persistent identifiers, the citation key does not need to include a prefix.
    Citing `@old-manuscript` will work, but only if reference metadata is [provided manually](#reference-metadata).
 
+Manubot is able to infer certain prefixes,
+such some citations can be formatted like `@accession` (without a prefix).
+Examples includes DOIs like `@10.15363/thinklab.4` or `@10/993`,
+PMC / PubMed identifiers like `@PMC4497619` or `@26158728`,
+arXiv identifier like `@1508.06576v2`,
```

Comment:
This documentation reads well.

The phrasing here could be "arXiv identifiers" but that can wait for a later pull request.